packages/maskbook/src/extension/options-page/DashboardComponents/TransferDialog.tsx

```diff
@@ -108,18 +110,10 @@ function TransferTab(props: TransferTabProps) {
         WalletMessages.events.transactionDialogUpdated,
         (ev) => {
             if (ev.open) return
-
-            // reset state
             resetTransferCallback()
-
             if (transferState.type !== TransactionStateType.CONFIRMED) return
-
-            // reset form
-            setAmount('')
-            setAddress('')
-            setMemo('')
-
-            // revalidate
+            onClose()
```

Comment:
Why prefers to close the dialog rather than reset it?

Reply:
No, it will be reset after close the dialog.